### PR TITLE
13838

### PR DESCRIPTION
**Pull request by ijobling**
In preparation for Content Authoring update -- more to do (update changelog for instance once date known)

### CODE REVIEW

File: source/instructors/authoring/assessments/edit-assessment.rst [hunk 1]
```diff
@@ -1,14 +1,19 @@
 .. meta::
-   :description: Edit an assessment by clicking on the settings icon and and clicking the assessment tab.
+   :description: Edit an assessment by clicking on the sdit button.
```
Comment (shajason) [inline source/instructors/authoring/assessments/edit-assessment.rst:2]: sdit>edit

File: source/instructors/authoring/guides/authoring_guidelines.rst [hunk 2]
```diff
@@ -12,7 +12,7 @@ Style your content using Markdown
 You can style your content using Markdown, a simple formatting language that is easier to use than HTML. You can embed HTML snippets in Guide content if you need a feature that is unavailable in Markdown.
 
 
-Load the **Demo Guides and Assessments** project to learn more about authoring with Guides
+Load the Demo Guides and Assessments project to learn more about authoring with Guides
```
Comment (shajason) [inline source/instructors/authoring/guides/authoring_guidelines.rst:15]: Wondering why you took the bold off - not a big deal either way

File: source/instructors/setupcourses/library/unlink-assessment.rst [hunk 3]
```diff
@@ -5,21 +5,25 @@
 
 Unlink an Assessment
 ====================
-If you want to modify an assessment that is the library, you can unlink it, make your changes, and then add your own version to your organization's assessments library. Follow these steps to unlink an assessment from the library:
+If you want to modify an assessment that is the library, you can unlink it, make your changes, and then add your own version to your organization's assessments library. Follow these steps to find the assessment you wish to modify, either:
```
Comment (shajason) [inline source/instructors/setupcourses/library/unlink-assessment.rst:8]: ...that is in the library (missing word in)

File: source/instructors/authoring/guides/authoring_guidelines.rst (hunk 2 above)
Comment (ijobling) [reply source/instructors/authoring/guides/authoring_guidelines.rst:15]: Just made the section title on deployed page look weird and looked better without: see https://docs.codio.com/instructors/authoring/guides/authoring_guidelines.html

**Review by shajason — DISMISSED** [review]

**Review by shajason — APPROVED** [review]

**Pull request merged**